### ISSUE TITLE
Can we add config options to include/exclude branches?

### ISSUE THREAD

**Issue by halvardssm** [pixta-dev/repository-mirroring-action#2]
First of all, I think this action is great, thank you for making it! 

As title suggests, I would love if we could have an option for include/exclude branches for mirroring. E.g. I only want to backup my master branch to gitlab whenever something gets merged/pushed to master in github.

**Comment by cipchk** [comment]
+1，For example: `gh-pages`

**Comment by yasaichi** [comment]
@halvardssm 
Sorry for the late reply. I'll try to implement your request.

**Comment by SecretsNetwork** [comment]
@yasaichi any updates?

**Comment by Fuseteam** [comment]
yes please, at the moment this action is deleting all branches other than master

**Comment by yasaichi** [comment]
@Fuseteam 
Which version do you use `actions/checkout`? If you use v2, you should specify `fetch-depth: 0` because it fetches only a single commit by default.

**Comment by Fuseteam** [comment]
I used version 1 iirc guess i need to updat— wait actions/checkout? Hmm i need to double check, thanks for the tip

**Comment by Fuseteam** [comment]
I think i __am__ using it with `actions/checkout`
